shell/platform/common/client_wrapper/basic_message_channel_unittests.cc

```diff
@@ -20,23 +21,39 @@ class TestBinaryMessenger : public BinaryMessenger {
   void Send(const std::string& channel,
             const uint8_t* message,
             const size_t message_size,
-            BinaryReply reply) const override {}
+            BinaryReply reply) const override {
+    send_called_ = true;
+    int length = static_cast<int>(message_size);
+    last_message_size_ = length;
+    std::vector<uint8_t> last_message(length);
+    memcpy(&last_message[0], &message[0], length * sizeof(uint8_t));
+    last_message_ = last_message;
```

Comment:
Can we do something like this?

```suggestion
    last_message_ = std::vector<uint8_t>(message, message + length * sizeof(uint8_t));
```

Reply:
Agreed, this doesn't look preferred.